ecl/hql/hqlexpr.cpp

```diff
@@ -15590,7 +15596,7 @@ IHqlExpression * convertToSimpleAggregate(IHqlExpression * expr)
     loop
     {
         node_operator op = ds->getOperator();
-        if ((op != no_sorted) && (op != no_distributed) && (op != no_preservemeta) && (op != no_alias_scope))
+        if ((op != no_sorted) && (op != no_distributed) && (op != no_unordered) && (op != no_preservemeta) && (op != no_alias_scope))
```

Comment:
no it is correct - it is skipping operators which only provide meta information.